Source/WebCore/rendering/ReferencedSVGResources.cpp

```diff
@@ -272,10 +269,11 @@ RefPtr<SVGElement> ReferencedSVGResources::referencedPaintServerElement(TreeScop
     return elementForResourceIDs(treeScope, resourceID, { SVGNames::linearGradientTag, SVGNames::radialGradientTag, SVGNames::patternTag });
 }
 
-RefPtr<SVGFilterElement> ReferencedSVGResources::referencedFilterElement(TreeScope& treeScope, const ReferenceFilterOperation& referenceFilter)
+RefPtr<SVGFilterElement> ReferencedSVGResources::referencedFilterElement(TreeScope& treeScope, const Style::ReferenceFilterOperation& referenceFilter)
 {
     if (referenceFilter.fragment().isEmpty())
         return nullptr;
+
     RefPtr element = elementForResourceID(treeScope, referenceFilter.fragment(), SVGNames::filterTag);
     return element ? downcast<SVGFilterElement>(WTFMove(element)) : nullptr;
```

Comment:
Pretty sure downcast with `RefPtr` already handles null. So this does the same thing and we should consider switching to it.
```suggestion
    return downcast<SVGFilterElement>(elementForResourceID(treeScope, referenceFilter.fragment(), SVGNames::filterTag));
```